setup.sh

```diff
@@ -5,17 +5,14 @@
 
 set -e
 
+# Docker environment which we will need to install for docker-osx-dev to work.
+export   DOCKER_HOST="tcp://localhost:2375"
```

Comment:
Ah, fair enough. Probably worth mentioning that in a comment so it's clear why it's different than the other variables in the script.